fixtures/ios_app_with_shared_mergeable_framework/Project.swift

```diff
@@ -0,0 +1,55 @@
+import ProjectDescription
+
+let project = Project(
+    name: "App",
+    targets: [
+        Target(
+            name: "App",
+            destinations: .iOS,
+            product: .app,
+            bundleId: "io.tuist.App",
+            infoPlist: "App/Info.plist",
+            sources: ["App/Sources/**"],
+            dependencies: [
+                .project(target: "SharedDependenciesFramework", path: "Modules/SharedDependenciesFramework"),
```

Comment:
Hey @kwridan ,

Even though I suggest that this is an Apple's bug that mergeable only works in a single project, you are right. I'd change my example to avoid creating multiple **xcodeproj** files.

> Seeing the app directly imports DynamicFrameworkA and DynamicFrameworkB would we not need to be explicit about depending on them too? - I can see the SharedDependenciesFramework combines those.

Based on how it works, we only have to link and embed `SharedDependenciesFramework`. However, we should still use `import DynamicFrameworkA` and `import DynamicFrameworkB` in our codebase.

That's the point of this shared framework - it wraps all your dependencies and includes them in your code base either statically or dynamically, so you don't have to every time go into all of your targets and add your newly created frameworks (unless they are not mergeable).